static/app/views/traces/content.tsx

```diff
@@ -39,7 +39,7 @@ const DEFAULT_PER_PAGE = 50;
 export default function Wrapper(props) {
   const organization = useOrganization();
 
-  if (organization.features.includes('xvisibility-explore-view')) {
+  if (organization.features.includes('visibility-explore-view')) {
```

Comment:
Was this meant to be part of these changes?

Reply:
Yes, just a typo that I had and doesn't affect any customers